src/main/java/org/jabref/gui/entryeditor/SciteTab.java

```diff
@@ -97,7 +97,14 @@ private VBox getTalliesPane(SciteTallyModel tallModel) {
         Label titleLabel = new Label(Localization.lang("Tallies for %0", tallModel.doi()));
         titleLabel.getStyleClass().add("scite-tallies-label");
         Text message = new Text(Localization.lang(
-                "Total Citations: %0\nSupporting: %1\nContradicting: %2\nMentioning: %3\nUnclassified: %4\nCiting Publications: %5",
+                """
+                Total Citations: %0
```

Comment:
Uh this is not a good idea, I don't think our localization code can handle this. Please revert and ignore the bot

Reply:
Yes, I have reverted this in [feb3cb7](https://github.com/JabRef/jabref/pull/12883/commits/feb3cb7de128bac40050d2a987bfdb4e2ca0a99a).